roles/k3s_server_post/defaults/main.yml

```diff
@@ -30,3 +30,4 @@ metal_lb_mode: layer2
 metal_lb_available_timeout: 240s
 metal_lb_controller_tag_version: v0.14.3
 metal_lb_ip_range: 192.168.30.80-192.168.30.90
+metal_lb_interfaces:
```

Comment:
Should this be an empty list by default?

Reply:
@kuboschek

Reply:
This way I can have a condition on this in metallb.crs.j2. If empty, no interfaces are set in the L2Advertisement resource, which in turn triggers the default of metallb: https://metallb.universe.tf/configuration/_advanced_l2_configuration/#specify-network-interfaces-that-lb-ip-can-be-announced-from

Let me know if there's a better default and I'll adjust.